src/manifest-chrome.json

```diff
@@ -17,7 +17,8 @@
   },
   "permissions": [
     "tabs",
-    "activeTab"
+    "activeTab",
+    "storage"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**🎯 suggestion(non-blocking):** Peux tu réordonner par ordre alphabétique ? :)